Police-Data-Accessibility-Project/data-sources-app-v2
# refactor: miscellaneous client architectural updates

This PR adds the following architectural updates to the client in advance of the v2 UI:

- Adds file-based routing with [unplugin-vue-router](https://uvr.esm.is/)
- Updates tests (removes page-level tests, anticipating using integration tests rather than unit tests to cover those components).
- Adds subtle fade-in/fade-out route transitions 

## To test
1. Run the API
2. Run the client
3. Ensure everything works as previously (no need to thoroughly test the views, as most of them will be changing).
